src/services/CatfishService/index.ts

```diff
@@ -19,13 +19,10 @@ export class CatfishService {
       headers,
     );
 
-    if (!response.ok) {
-      // check if catfish token has expired
-      if (response.status === 401) {
-        // ask user to connect with fractal wallet again
-        this.fractalAccount.clearTokens();
-        throw ERRORS_CATFISH_TOKEN_EXPIRED();
-      }
+    if (!response.ok && response.status === 401) {
```

Comment:
If `.ok` is equivalent to `.status` returning `20X`, we could do with the second condition only